servers/rendering_server.cpp

```diff
@@ -1399,7 +1397,7 @@ Array RenderingServer::_get_array_from_surface(uint64_t p_format, Vector<uint8_t
 								tangentsw[j * 4 + 3] = tan.w;
 							}
 							ret[RS::ARRAY_NORMAL] = normals;
-							ret[RS::ARRAY_FORMAT_TANGENT] = tangents;
+							ret[RS::ARRAY_TANGENT] = tangents;
```

Comment:
This alone fixes https://github.com/godotengine/godot/issues/83526. What a silly mistake